contents/why_julia.md

```diff
@@ -21,15 +21,15 @@ Out of the blue you hear a name: "Julia".
 What is this?
 How is it any different from other stuff that I've been told that you ought use to do data science?
 
-Why should I dedicate my precious time into learning a language that is almost never mentioned in any job listing, lab position, postdoc offer or academic job description?
+Why should I dedicate my precious time into learning a language that is almost never mentioned in any job listing, lab position, postdoc offer or academic job description?. Being slowly less common as the language is gaining reputation. 
```

Comment:
```suggestion
Why should I dedicate my precious time into learning a language that is almost never mentioned in any job listing, lab position, postdoc offer or academic job description?. Being slowly less common as the language is gaining reputation. 
```

I don´t want to associate Julia with the word "slow" could we change?

Reply:
Sure, the idea is that. Despite the cons you mentioned. The language is gaining reputation, hence more job openings are also asking for Julia. I want to say that there is future here. How would you said it?

Reply:
Why not "gaining traction"

Reply:
I mean I agree with the argument, I just want to get rid of the word "slowly". I would like a more positive one.